compiler/rustc_middle/src/ty/layout.rs

```diff
@@ -324,6 +324,7 @@ impl<'tcx> SizeSkeleton<'tcx> {
         param_env: ty::ParamEnv<'tcx>,
     ) -> Result<SizeSkeleton<'tcx>, &'tcx LayoutError<'tcx>> {
         debug_assert!(!ty.has_non_region_infer());
+        let param_env = param_env.with_reveal_all_normalized(tcx);
```

Comment:
Ok yeah, this preserves the revealing behavior of size computation. I think that's why more tests don't explode, makes sense.

Reply:
yea, I had lots of tests explode, and didn't look too deeply, but assumed I could write a stable test that would break